ui/src/pages/frontsite/components/Document/DocumentLinks.js

```diff
@@ -21,7 +21,7 @@ export class DocumentLinks extends Component {
 
   hasPermissions = eitem => {
     if (!eitem.open_access) {
-      return sessionManager.authenticated;
+      return sessionManager.isAuthenticated();
```

Comment:
thanks for fixing this, I have no idea why it was working for me, I will check it